### PR TITLE
Add COTM Plugin

### PR DESCRIPTION
**Pull request by guanbinrui**
A quick plugin for composing NFT tokens in #CreativityOnTheMove - Strange Design meeting-up.

+ [x] Scaffolding
+ [x] UI/UX
+ [ ] Deploy the token contract
+ [x] Mint on the chain

### CODE REVIEW

File: packages/maskbook/src/plugins/COTM/assets/Lisa.ts [hunk 1]
```diff
@@ -0,0 +1 @@
...velyYJPFldJ9xQVelVb5h/T4EvuJSolVoTQKQxl4moURjFzNwwQ/gM1+MK8OfC77hKnypuICXsJNxKY+g+2IOJvtwud6z1z1fNIfQkcaibdWe7XCBXybMdCEu2F/5fOBCNa5yd3HO3OPum3fGu/ZinA+cpeWM59zmnB6c8XPvPnCV8+V63t3H2CkuziLOshl3X4IvMkE9y+bJBNxbjM988d1VaIVneW1V+aPyxcKW+TJf/phxxlsmGPFFglnyuzqwGS0nXbUpM6nccMSnD8R2tOMTWDaczZs0lzTAfuu+fUWhQJRx8akFNsBlRz2PhXm0vQ/t7BiRXyR/zXZfSbAVV64EfyoVa8qUTGmOe/DFxU4TrpUOKnzgPNZRJA1n4Dq0mhuz68eyv/Ppi/mkUwefkhtMi5VxHmu+kFvGc2u+pKSIRyq3wUZLuMBN/3HnZH6mC5ToI/eR50Z8QfVsv6ePf4ow5Xxj9htr35+qKDmROaoowCdF3xXZi9o8/PGEq+GM5KrCJX8RmST4XKpcWDNVhD+ZSwchpLmPsSubYJ7IjEY70YFItJf7iPIarJchX5jLlKSYdxrwRWszrxf5pnwkPxQ+geUZOPp3jsheL8UJ9o7N3kyL183Zn89bzC0cHr9q5sc//k6tJfr9to39rglff/uPP7b2p+//9u9//P41pp0G/10zV37XTJy012tT5utm/nyNuZNGQAAHCibVP/yWZ+qT+eJybebN2/YMGiZQcN0SIPLbL1v76vsVfmccXmiBv/pCfY3eWwRB5qbBQD9t1Uzd0M/F2c/3pZz38TPqHddl2gq/KXJeqQZe9idzmaOUyHRXOZEQ0lImea+Xr8OhHkFEPkNhMoQW7fnwu+ITwQlak6M/vmGOaLR5kUCElUsXJVyMzbheDnBdrHE9D/jgC352gct8MX5a8KXDZRPvadDK50a+HAsXUa1R0aHDhXnSfL3QHMJ3+rIpExq7ShMraemYT48Nc3J7psfutr3lH2LAGd71FneVTDaZMk/CX5VcZjLxEumZEpZK43EUzvf4PI3vPnB9sFPOeBzYe18wn/E+H/i80twSzRk+bu07uBAy3n7GP/UZT/Rluk90lm3ZvMknuId3X5UJfJbZH0z30Qa+WHjqccXdh3mxygT9PQ2uIhNYQ3UUSXbx7xvcfdkcTXOpqSP84bP8MZ8v9Yw3X4Cz/AE
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/COTM/assets/Lisa.ts:1]: Why not store it as a picture?

Comment (guanbinrui) [reply packages/maskbook/src/plugins/COTM/assets/Lisa.ts:1]: The CORS strategy prohibits the image from `/public` dir.

**Comment by guanbinrui** [comment]
@Jack-Works @septs  This PR is a clone of Election 2020. Have a quick check and we have to merge it ASAP.

File: packages/maskbook/src/plugins/COTM/define.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+const [COTM_CompositionEntry, COTM_CompositionUI] = createCompositionDialog('👽 #CreativityOnTheMove', (props) => (
+    <COTM_CompositionDialog {...props} />
+))
+
+export const COTM_PluginDefine: PluginConfig = {
+    pluginName: 'COTM',
+    identifier: COTM_PluginID,
+    stage: PluginStage.Production,
+    scope: PluginScope.Public,
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/COTM/define.tsx:21]: Should it be public?

File: packages/maskbook/src/plugins/COTM/define.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    successDecryptionInspector: function Comp(props) {
+        const payload = COTM_MetadataReader(props.message.meta)
+        if (!payload.ok) return null
+        return (
+            <MaskbookPluginWrapper pluginName="🇳🇱 #CreativityOnTheMove">
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/COTM/define.tsx:26]: What is this "NL"?

**Review by Jack-Works — APPROVED** [review]

File: packages/maskbook/src/plugins/COTM/assets/Lisa.ts (hunk 1 above)
Comment (Jack-Works) [reply packages/maskbook/src/plugins/COTM/assets/Lisa.ts:1]: So you should use the `<Image />` element in our codebase. It supports workaround CORS

**Pull request merged**